worker/sink_handler.go

```diff
@@ -86,10 +86,18 @@ func newKafkaSink(config *z.SuperFlag) (Sink, error) {
 	saramaConf.Producer.Return.Successes = true
 	saramaConf.Producer.Return.Errors = true
 
-	if config.GetPath("ca-cert") != "" {
-		tlsCfg := &tls.Config{
-			MinVersion: tls.VersionTLS12,
+	if config.GetBool("tls") && config.GetPath("ca-cert") == "" {
```

Comment:
What's the action if one sets `tls=true` but doesn't specify `ca-cert`?

Reply:
we just use the system certs